MergeKSortedLinkedList.py

```diff
@@ -0,0 +1,46 @@
+# TC = O(nlogk) -- n is avg length of linkedList
```

Comment:
N is the total number of elements in all the linkedlist
If you want to say it is average number of elements in each LL , then write it as nklogk